doc/dev/cephfs-mirroring.rst

```diff
@@ -22,6 +27,31 @@ This feature is tracked here: https://tracker.ceph.com/issues/47034
 .. note:: Synchronizing hardlinks is not supported -- hardlinked files get synchronized
           as separate files.
 
+Creating Users
+--------------
+
+Start by creating a user (on the primary/local cluster) for the mirror daemon. This user
+has restrictive capabilities on the MDS and the OSD::
+
+  $ ceph auth get-or-create client.mirror mon 'allow r' mds 'allow r' osd 'allow rw' mgr 'allow r'
+
+Create a user for each filesystem peer (on the secondary/remote cluster). This user needs
+to have full capabilities on the MDS (to take snapshots) and the OSDs::
+
+  $ ceph auth get-or-create client.mirror_remote mon 'allow r' mds 'allow rwps' osd 'allow rw' mgr 'allow r'
+
+This user should be used (as part of peer specification) when adding a peer.
+
+Starting Mirror Daemon
+----------------------
+
+Right now, mirror daemons need to be spawned manually. This will be replaced by providing
+`systemctl(1)` command to start/stop mirror daemons. For now, use::
```

Comment:
are the systemd units in this PR not usable currently?

Reply:
They can be use (but needs review) -- the doc needs updation. I'll push an update.